pkg/kubelet/prober/worker.go

```diff
@@ -316,6 +325,29 @@ func (w *worker) doProbe(ctx context.Context) (keepGoing bool) {
 		return true
 	}
 
+	if resultChanged {
+		var eventType string
+		var eventReason string
+		if result == results.Success {
+			eventType = v1.EventTypeNormal
+			eventReason = events.ContainerProbeResultSuccess
```

Comment:
Could you elaborate on the benefits of exposing Events for probe success? Having 1 event per container is pretty expensive and considering that probe success is not actionable, I don't it is worth it.

Reply:
it is for the switch from failed to success. It is handy for troubleshooting why something was not working and started working again. Or getting a timestamp when it supposed to be turned back to working. Mostly a quality of life event for end users

Reply:
Make sense :+1: